plugin-scalebar/src/test/java/com/mapbox/maps/plugin/scalebar/ScaleBarImplTest.kt

```diff
@@ -140,4 +145,44 @@ class ScaleBarImplTest {
     assertTrue(scaleBarView.refreshHandler.hasMessages(MSG_RENDER_CONTINUOUS))
     Shadows.shadowOf(Looper.getMainLooper()).idle()
   }
+
+  @Test
+  fun `verify correct scale bar segments returned`() {
+    // First verify for metric
+    scaleBarSettings.isMetricUnits = true
+    scaleBarView.settings = scaleBarSettings
+    `verify below and above scale bar segments`(metricTable)
+    // Then verify for imperial units
+    scaleBarSettings.isMetricUnits = false
+    scaleBarView.settings = scaleBarSettings
+    `verify below and above scale bar segments`(imperialTable)
+  }
+
+  private fun `verify below and above scale bar segments`(scaleTable: List<Pair<Int, Int>>) {
+    // Special case where max distance is smaller than the first entry segment in the table
+    val belowFirstUnitBarDistance = scaleTable[0].first.toFloat() / scaleTable[0].second - 0.1F
```

Comment:
nit:
just for clarity
```suggestion
    val belowFirstUnitBarDistance = (scaleTable[0].first.toFloat() / scaleTable[0].second) - 0.1F
```